pages/Blogs/index.tsx

```diff
@@ -3,16 +3,32 @@ import { FC, useEffect, useState } from 'react';
 import BlogHighlightSection from '@components/Blogs/BlogHighlightSection';
 import BlogListSection from '@components/Blogs/BlogListSection';
 import Filter from '@components/Common/Filter';
-import { getAllBlogCategories, getAllBlogs } from '@store/blogSlice';
+import { getAllBlogs } from '@store/blogSlice';
 import { useAppDispatch, useAppSelector } from '@store/hooks';
 import { removeElement } from '@utils/helper';
 import Head from 'next/head';
 import _ from 'underscore';
 
 const Blogs: FC = () => {
   const dispatch = useAppDispatch();
-  const { categories, blogs } = useAppSelector((state) => state.blogs);
-  const optionNames = ['All', ...categories, 'Other'];
+  const { blogs } = useAppSelector((state) => state.blogs);
+  const optionNames = [
```

Comment:
can we show the options only if they exist in the list of data, feels overwhelming w all the options

Reply:
Fixing rn